cpu/cortexm_common/include/cpu.h

```diff
@@ -36,6 +36,7 @@
 #include "sched.h"
 #include "thread.h"
 #include "cpu_conf.h"
+#include "bitband.h"
```

Comment:
but doesn't this contradict the 'import only what is neccessary' thing that has impacts to the build process/speed?!